cpp/inc/bond/core/nullable.h

```diff
@@ -501,7 +505,13 @@ class nullable<T, Allocator, false>
         T* p = alloc.allocate(1);
         try
         {
-            return ::new(static_cast<void*>(p)) T(std::forward<Arg1>(arg1));
+#ifndef BOND_NO_CXX11_ALLOCATOR
+            std::allocator_traits<AllocatorT>::construct(alloc, p,
```

Comment:
Minor: let's put the arguments either all on the same line or all on different lines.